index.bs

```diff
@@ -11632,20 +11633,23 @@ is the concatenation of the [=interface=]’s
 
 <div algorithm="to invoke the [[DefineOwnProperty]] internal method of named properties object">
 
-    When the \[[DefineOwnProperty]] internal method of a [=named properties object=] is called,
+    When the \[[DefineOwnProperty]] internal method of a [=named properties object=] |O|
+    is called with property key |P| and [=Property Descriptor=] |Desc|,
     the following steps are taken:
 
-    1.  Return <emu-val>false</emu-val>.
+    1.  If [$Type$](|P|) is String, then return <emu-val>false</emu-val>.
+    1.  Otherwise, return [=?=] [$OrdinaryDefineOwnProperty$](|O|, |P|, |Desc|).
 </div>
 
 <h5 id="named-properties-object-delete">\[[Delete]]</h5>
 
 <div algorithm="to invoke the [[Delete]] internal method of named properties object">
 
-    When the \[[Delete]] internal method of a [=named properties object=] is called,
-    the following steps are taken:
+    When the \[[Delete]] internal method of a [=named properties object=] |O|
+    is called with property key |P|, the following steps are taken:
 
-    1.  Return <emu-val>false</emu-val>.
+    1.  If [$Type$](|P|) is String, then return <emu-val>false</emu-val>.
+    1.  Otherwise, return [=?=] [$OrdinaryDelete$](|O|, |P|).
```

Comment:
Unlike in [legacy platform objects](https://heycam.github.io/webidl/#legacy-platform-object-delete), we can call [`OrdinaryDelete`](https://tc39.es/ecma262/#sec-ordinarydelete) instead of reimplementing it, as it's reachable only for own symbol properties. I mildly prefer calling ordinary methods so the implementations can match the spec more closely.